clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -470,6 +471,19 @@ public void testExplicitlyEnableJmxReporter() {
         admin.close();
     }
 
+    @Test
+    public void testExplicitlyEnableTelemetryReporter() {
```

Comment:
Should we add a "negative test" to ensure that the telementry-reported is not added by default?

Reply:
Did you miss this one?